### ISSUE TITLE
Page deletion confirmation text lacks sufficient contrast

### ISSUE THREAD

**Issue by chosak** [wagtail/wagtail#12232]
### Issue Summary

When deleting a page that triggers the [`WAGTAILADMIN_UNSAFE_PAGE_DELETION_LIMIT`](https://docs.wagtail.org/en/stable/reference/settings.html#wagtailadmin-unsafe-page-deletion-limit) check, the orange warning text fails contrast checks:

<img width="323" alt="image" src="https://github.com/user-attachments/assets/f3739b62-e6f4-4c2b-8571-011ec26c90ed">

The foreground color is #FAA500 and the background is white (#FFFFFF). This fails Chrome's Lighthouse check with the error message "Background and foreground colors do not have a sufficient contrast ratio." Additionally it fails the check on [the WebAIM Contrast Checker](https://webaim.org/resources/contrastchecker/):

<img width="432" alt="image" src="https://github.com/user-attachments/assets/41341cea-bc5d-482e-ae3e-650ac0ddcadc">

### Steps to Reproduce

1. Start a new project with `wagtail start myproject`, then `cd myproject`
2. Alter myproject/settings.py to set `WAGTAILADMIN_UNSAFE_PAGE_DELETION_LIMIT = 1` (to make this easier to test)
3. `./manage.py migrate && ./manage.py createsuperuser && ./manage.py runserver`
4. Add a child page to the home page, then try deleting the home page.

The displayed warning text ("This action will delete total 2 pages") fails the accessibility contrast check.

-   I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: yes

### Technical details

-   Python version: 3.11.1
-   Django version: 5.1
-   Wagtail version: 6.2
-   Browser version: Chrome 126

### Working on this

Anyone can contribute to this. View our [contributing guidelines](https://docs.wagtail.org/en/latest/contributing/index.html), add a comment to the issue once you’re ready to start.

**Comment by imavisingh03** [comment]
I would love to work on improving the contrast ratio of the warning text to meet accessibility standards. I’ll update the issue once I have a pull request ready!

**Comment by sanjeevholla26** [comment]
Hello! 
I want to start contributing to wagtail. So can I work on this issue if no one else is working on it?
